internal/services/executor/driver/docker_test.go

```diff
@@ -450,4 +450,96 @@ func TestDockerPod(t *testing.T) {
 			t.Fatalf("unexpected exit code: %d", code)
 		}
 	})
+
```

Comment:
Can you adapt the previous volume tests grep syntax to the one you used in the new tests?